ACS/ACS/MainWindow.xaml.cs

```diff
@@ -2610,8 +2610,9 @@ private void OpenBrowserHelp(String relativeHelpPath, String queryString)
             }
             else
             {
-                String absoluteURL = new Uri(Environment.CurrentDirectory + "/" + relativeHelpPath).AbsoluteUri + queryString;
-                logger.Log(LogLevel.Debug, "ACS is not connected, so starting local help with URL " + absoluteURL);
+                String baseURL = ini.IniReadValue("Options", "acsOnlineHelpBaseURL");
+                String absoluteURL = baseURL + relativeHelpPath + queryString;
+                logger.Log(LogLevel.Debug, "ACS is not connected, so starting online help with URL " + absoluteURL);
                 Process.Start(browserPath, absoluteURL);
```

Comment:
Please show error message, if helpBrowserCommand can not be executed.
Tell user that he can configure the browser command in asterics.ini